.gitlab-ci.yml

```diff
@@ -15,6 +15,13 @@ build:
   script:
     - ./gradlew build -x :smoke-tests:test -x :testing:profiler-tests:test --scan --no-daemon --stacktrace
 
+snapshot:
+  stage: release
+  rules:
+    - if: '$CI_COMMIT_REF_NAME == "main"'
+  script:
+    - ./gradlew snapshot --no-daemon --stacktrace
```

Comment:
This does not need repository closing and releasing and whatnot?

Reply:
I think it doesn't: staging, closing and releasing are just for non-snapshot versions.